.github/workflows/dev-packages.yaml

```diff
@@ -0,0 +1,36 @@
+# Action to publish packages under the `next` tag for testing
+# Packages are versioned as `0.0.0-dev.{short-sha}`
```

Comment:
Is this comment still accurate? Changesets version pattern is slightly different

Reply:
Good catch! This has been updated to the new format